### ISSUE TITLE
supervisord no such file or directory

### ISSUE THREAD

**Issue by 1josh13** [milleruk/alliance_auth_docker#7]
Hello, 

I am getting the error below when launching from docker-compose after it builds the image. This happens on both my server as well as my local testing env. 

Thanks

`ERROR: for allianceauth  Cannot start service allianceauth: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"/usr/bin/supervisord\": stat /usr/bin/supervisord: no such file or directory": unknown`

**Comment by vocino** [comment]
I'm having the same issue. Did you ever figure this out?

**Comment by notepid** [comment]
looks like the installation location of supervisord has changed. I got it running by changing the supervisord location and the path to the configuration file. This my Dockerfile :

```
FROM python:3.7.7-buster
RUN apt-get update && apt-get upgrade -y && apt-get install -y \
    libmariadb-dev \
    unzip \
    git \
    curl \
    libssl-dev \
    libbz2-dev \
    libffi-dev
RUN python3 -m venv /home/allianceserver/venv/auth/
#RUN source /home/allianceserver/venv/auth/bin/activate
RUN pip install --upgrade pip
RUN pip install wheel gunicorn flower
RUN pip install allianceauth
RUN cd /home/allianceserver && allianceauth start myauth
COPY /conf/local.py /home/allianceserver/myauth/myauth/settings/local.py
RUN cd /home/allianceserver && allianceauth update myauth
RUN mkdir -p /var/www/myauth/static
RUN python /home/allianceserver/myauth/manage.py collectstatic
RUN pip install supervisor
RUN pip install -U git+https://github.com/shaib/supervisor-stdout.git@patch-1
COPY /conf/supervisord.conf /usr/local/etc/supervisord.conf
WORKDIR /home/allianceserver/myauth
EXPOSE 8000
EXPOSE 5555
CMD ["/usr/local/bin/supervisord"]
```

**Comment by milleruk** [comment]
Fixed with commit 5cc7e46e43559fa209329e4010509b84fb9bbb62
